javascript/javascript1/week2/candystore.js

```diff
@@ -0,0 +1,49 @@
+const candyPriceTable = {
+  sweet: 0.5,
+  chocolate: 0.7,
+  toffee: 1.1,
+  chewing_gum: 0.03,
+};
+const boughtCandyPrices = [];
+let totalPrice = 0;
+function addCandy(candyType, weight) {
+  if (candyType === "sweet") {
+    priceOfCandy = weight * candyPriceTable.sweet;
+    console.log("sweet" + priceOfCandy);
+  } else if (candyType === "chocolate") {
+    priceOfCandy = weight * candyPriceTable.chocolate;
+  } else if (candyType === "toffee") {
+    priceOfCandy = weight * candyPriceTable.toffee;
+  } else if (candyType === "chewing-gum") {
+    priceOfCandy = weight * candyPriceTable.toffee;
```

Comment:
it is a wrong candy type you are using here

Reply:
sorry..when copy/paste mistake